addons/mail/static/tests/discuss_app/im_status.test.js

```diff
@@ -59,7 +59,7 @@ test("initially away", async () => {
 });
 
 test("change icon on change partner im_status", async () => {
-    patchWithCleanup(Persona, { IM_STATUS_DEBOUNCE_DELAY: 0 });
+    patchWithCleanup(ResPartner, { IM_STATUS_DEBOUNCE_DELAY: 0 });
```

Comment:
`IM_STATUS_DEBOUNCE_DELAY` should be moved to a common model (I guess store if nothing else).

Reply:
I moved it to the Store class.